controllers/apps/cluster_controller.go

```diff
@@ -172,6 +172,8 @@ func (r *ClusterReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ct
 			&FillClassTransformer{},
 			// create cluster connection credential secret object
 			&ClusterCredentialTransformer{},
+			// handle rbac for pod
```

Comment:
missing SA existence validation

Reply:
in order to do validation we need Read-Only (get,list,watch) access for SA, Role, RoleBinding kind

Reply:
it is checked in ClusterCredentialTransformer.transform

Reply:
sorry, it is in controllers/apps/transformer_rbac.go